eva-server/src/main/java/uk/ac/ebi/eva/server/ws/ga4gh/beaconv2/DefaultApi.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * European Variation Archive (EVA) - Open-access database of all types of genetic
+ * variation data from all species
+ *
+ * Copyright 2019 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * NOTE: This class is auto generated by the swagger code generator program (3.0.8).
+ * https://github.com/swagger-api/swagger-codegen
+ * Do not edit the class manually.
+ */
+package uk.ac.ebi.eva.server.ws.ga4gh.beaconv2;
+
+import uk.ac.ebi.eva.commons.beacon.models.Beacon;
+import io.swagger.annotations.*;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RequestPart;
+import org.springframework.web.multipart.MultipartFile;
+
+import javax.validation.Valid;
+import javax.validation.constraints.*;
+import java.util.List;
+import java.util.Map;
+@javax.annotation.Generated(value = "io.swagger.codegen.v3.generators.java.SpringCodegen", date = "2019-06-18T18:08:34.969Z[GMT]")
+@Api(value = "default", description = "the default API")
+public interface DefaultApi {
+
+    @ApiOperation(value = "", nickname = "getBeacon", notes = "Get information about the beacon", response = Beacon.class, tags={  })
+    @ApiResponses(value = { 
+        @ApiResponse(code = 200, message = "successful operation", response = Beacon.class) })
+    @RequestMapping(value = "/",
+        produces = { "application/json" }, 
+        method = RequestMethod.GET)
+    ResponseEntity<Beacon> getBeacon();
```

Comment:
I am sorry, I didn't understand what was meant

Reply:
not really. I was waiting to merge to try out a working version to provide this kind of feedback

Reply:
@junaidnz97, I mean that if someone has tested the functionality itself. It will be unfortunate to create a lot of Swagger annotations and find out after merging that they don't work.

Reply:
@cyenyxe Oh, okay :+1:

Reply:
@junaidnz97 Once you have checked that the information is displayed, could you please send us a screenshot?